docs/core-packages/generator.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: docs
+title: babel-generator
+description: Turns an AST into code.
+permalink: /docs/core-packages/generator/
+package: babel-generator
+---
+
+{% include package_readme.html %}
```

Comment:
The options table is broken, I will look at it.

![screenshot from 2017-03-16 08-38-16](https://cloud.githubusercontent.com/assets/1493671/23986343/ec605a92-0a23-11e7-8564-51834bba0b84.png)